src/data/real/ennreal.lean

```diff
@@ -150,18 +216,47 @@ coe_one ▸ coe_two ▸ by exact_mod_cast (@one_lt_two ℕ _ _)
 lemma two_ne_zero : (2:ennreal) ≠ 0 := (ne_of_lt zero_lt_two).symm
 lemma two_ne_top : (2:ennreal) ≠ ∞ := coe_two ▸ coe_ne_top
 
+/-- The set of `ennreal` numbers that are not equal to `∞` is equivalent to `ℝ≥0`. -/
+def ne_top_equiv_nnreal : {a | a ≠ ∞} ≃ ℝ≥0 :=
```

Comment:
Can you use this ito simplify `ennreal.ne_top_homeomorph_nnreal` in `topology.instances.ennreal`?

Reply:
See below.

Reply:
Oh, I missed that. Great!